### PR TITLE
Added workflow badges

### PR DESCRIPTION
**Pull request by a-velasco**
## Issue
This repo's status is not displayed in the [Charm Engineering Releases](https://releases.juju.is/?team=Data) overview.

## Solution
Added workflow badges for

- CharmHub Badge
- Release
- Tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Redis Operator
+[![Charmhub](https://charmhub.io/redis-k8s/badge.svg)](https://charmhub.io/redis-k8s)
+[![Release to latest/edge](https://github.com/canonical/redis-k8s-operator/actions/workflows/release.yaml/badge.svg)](https://github.com/canonical/redis-k8s-operator/actions/workflows/release.yaml)
+[![Tests](https://github.com/canonical/redis-k8s-operator/actions/workflows/ci.yaml/badge.svg)](https://github.com/canonical/redis-k8s-operator/actions/workflows/ci.yaml)
```
Comment (deusebio) [inline README.md:4]: Should we pin the branch as we do in other charms (e.g. [Kafka](https://github.com/canonical/kafka-operator/blob/main/README.md?plain=1#L5)]?

**Review by deusebio — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (a-velasco) [reply README.md:4]: As far as I understand, the only reason to specify a branch is if we want to display the badge for a branch other than the default. Is that the case for this operator?

**Review by zmraul — APPROVED** [review]
Looks good. #80 will need to be merged before this one.

**Comment by zmraul** [comment]
@a-velasco A rebase on main should have the tests passing here.

**Comment by a-velasco** [comment]
@zmraul thank you!

**Pull request merged**